endpoints-skeleton-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -33,14 +33,16 @@
     <fileSet filtered="true" encoding="UTF-8">
       <directory></directory>
       <includes>
-        <include>nbactions.xml</include>
-        <include>README.md</include>
+        <include>pom.xml</include>
       </includes>
     </fileSet>
     <fileSet encoding="UTF-8">
       <directory></directory>
       <includes>
         <include>LICENSE</include>
+        <include>README.md</include>
+        <include>nbactions.xml</include>
+        <include>.gitignore</include>
```

Comment:
It was in one of the templates, so I propagated it everywhere.  I was trying to make them the same on non-substantive issues.